pkg/localmetrics/localmetrics.go

```diff
@@ -107,7 +106,7 @@ func NewMetricsCollector(store cache.Cache) *MetricsCollector {
 			Name:        "aws_account_operator_account_claim_ready_duration_seconds",
 			Help:        "The duration for account claim cr to get claimed",
 			ConstLabels: prometheus.Labels{"name": operatorName},
-			Buckets:     accountDurationBuckets,
+			Buckets:     []float64{1, 5, 10, 20, 30, 45, 60},
```

Comment:
Considering this is in seconds, I wonder if we should add another bucket for the extreme cases where it takes >1min.

Something like `120`.

Reply:
That makes sense, will update it.